db/fixtures/chargeback_rate_detail_currencies.yml

```diff
@@ -1,21 +1,81 @@
 ---
-- :code: "USD"
-  :name: "Dollars"
-  :full_name: "United States Dollars"
-  :symbol: "$"
-  :unicode_hex: "36"
+- :code: "AUD"
+  :name: "Dollar"
+  :full_name: "Australian Dollar"
+  :symbol: "A$"
+- :code: "BRL"
+  :name: "Brazilian real"
+  :full_name: "Brazilian real"
+  :symbol: "R$"
+- :code: "CAD"
+  :name: "Dollar"
+  :full_name: "Canadian Dollar"
+  :symbol: "C$"
+- :code: "CHF"
+  :name: "Swiss franc"
+  :full_name: "Swiss franc"
+  :symbol: "Fr"
+- :code: "CNY"
+  :name: "Renminbi"
+  :full_name: "Renminbi"
+  :symbol: "元"
 - :code: "EUR"
   :name: "Euro"
   :full_name: "Euro"
   :symbol: "€"
-  :unicode_hex: "8364"
```

Comment:
Do we need a followup PR for the migration to drop this column?

Reply:
yes and also maybe more column. (not sure about usage of name)

about `unicode_hex` I am just thinking if we should  keep this column for any display reasons. (I did not meet any displaying issue of listed currency's symbols )